app/config/collections.php

```diff
@@ -825,10 +825,10 @@
             ],
             [
                 '$id' => 'updated',
```

Comment:
do we need this? `$updatedAt` should make this obsolete??

Reply:
Torsten and Eldad asked to leave this attribute